src/test/ceph_objectstore_tool.py

```diff
@@ -1796,16 +1797,12 @@ def main(argv):
         vstart(new=False)
         wait_for_health()
 
-        time.sleep(20)
```

Comment:
I unfortunately think there is a reason for that sleep but it's unclear what it is.